pkgs/development/tools/gomod2nix/update.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env nix-shell
```

Comment:
I would have done this script in bash because we are just shelling out to commands but fine for me.

Reply:
agreed. there is some JSON that's handled and a tiny bit of JSON creation but nothing that isn't trivial with `jq`

although I'm less inclined to keep it python personally

Reply:
I've ported the update script to bash and it's now using gomod2nix to update itself.